index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" href="styles.css" />
+        <link
+            rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+        />
+    </head>
+
+    <body class="container">
+        <header class="header">
+            <div class="menu-logo">
+                <img
+                    src="images/mobile_menu 1.svg"
+                    alt=""
+                    onclick="popMenu()"
+                />
+                <img src="images/logo.svg" alt="" class="header-logo" />
```

Comment:
Great design @Developer-Kingz 🤝

- [ ] Kindly ensure your mobile views don't have this logo as it is not given in the design guidelines.
- Your design
![logo mobile view](https://github.com/Developer-Kingz/Microverse-capstone/assets/63917725/6e95980d-5676-4c97-ab9d-88a4f0abe89e)
- Given design
![Given mobile](https://github.com/Developer-Kingz/Microverse-capstone/assets/63917725/bca7b5b3-15d9-4fda-81f4-02e4fcc4bdd3)